python/cudf/cudf/tests/test_avro_reader_fastavro_integration.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
I'm not sure about file name. It implies that we'll have a separate file for potential AVRO writer tests, and for unit tests too. It does not seem practical to me, but I might be wrong here. 
I can see how having separate files for readers and writers would make sense in the work flow (and we might watn to split existing tests), but not for unit/integration.

Reply:
Looks like people don't mind separate test files in general, and we'll likely split tests for different components into separate directories so this file name is fine.
I think the two tests with empty input (left a comment there) need to be kept in the original file (no comparison with fastavro there).

Reply:
all of the tests are integration tests because the depend on fastavro to generate the file, even when the file contains no records or fields. :shrug:

Reply:
riight, forgot about where the data is coming from.